spec/rubocop/ast/instance_variable_node_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::AST::InstanceVariableNode do
```

Comment:
Would prefer if you'd follow what's laid out in the other specs, which are very focused unit tests for the extension itself. This has a lot of tangential test cases that makes the suite more confusing than it needs to be. 🙇

Reply:
I'm confused about how these are tangential. They cover issues I found as I was going through. I'm not sure how a smaller test suite would actually cover the extension properly.